Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+
+  config.vm.define :ubuntu1604 do |ubuntu1604|
+    ubuntu1604.vm.box = 'ffuenf/ubuntu-16.04-server-amd64' # https://atlas.hashicorp.com/ffuenf/boxes/ubuntu-16.04-server-amd64/
```

Comment:
any reason, why we are not using the official box? https://atlas.hashicorp.com/ubuntu